android/modules/ui/src/java/ti/modules/titanium/ui/TabGroupProxy.java

```diff
@@ -91,27 +91,24 @@ public int getTabIndex(TabProxy tabProxy)
 		return tabs.indexOf(tabProxy);
 	}
 
-	@Kroll.method
-	public void disableTabNavigation(Object params)
+	@Kroll.setProperty
+	public void setEnabled(boolean enabled)
```

Comment:
should we add a deprecation period here? I saw that you've put 12.7.0 as deprecated and removed in the doc. 
As it was added two years ago to have `animated` as a parameter this change will remove that functionality.

Reply:
@m1ga I think a deprecation period until 13.0.0 is fine in this PR for both platforms as @hansemannn suggested. I will make the further changes.

Reply:
@m1ga I brought back the `disableTabNavigation()`. Internally using new APIs to keep same effects.